maxim/logger/gemini/utils.py

```
@@ -49,10 +49,18 @@ def parse_chat_message(
                     pass
                 else:
                     request_messages.append(
-                        GenerationRequestMessage(
-                            role=role, content=GeminiUtils.parse_part_text(m) or ""
-                        )
+                        {
+                            "role": role,
+                            "content": GeminiUtils.parse_part_text(m) or "",
+                        }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)